src/seeds/factory.ts

```diff
@@ -21,5 +21,5 @@ export const createFakerUser = async (quantity: number) => {
     saveUser.push(user);
   }
 
-  await AppDataSource.manager.save(saveUser);
+  return await AppDataSource.manager.save(saveUser);
```

Comment:
quando você retorna a promise não precisa do await
```suggestion
  return AppDataSource.manager.save(saveUser);
```